modules/persistent-storage-csi-vsphere-disable-storage.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-disable-storage_{context}"]
+= Disabling storage on vSphere
```

Comment:
Should this title reflect disabling AND enabling as this seems to cover both functions? I may be misinterpreting something here.....